### PR TITLE
fix(labs): Make `TypedObject`, `EchoDatabase` and `SpaceProxy` not proxied by @deepsignal/react

### PR DESCRIPTION
**Pull request by mykola-vrmchk**
### Summary
🐛🚚♻️

This pull request fixes a proxying bug in `labs-app` by importing and assigning some classes to global properties, and refactors the `client` package to export the `SpaceProxy` class.

> _`SpaceProxy` moves_
> _from `client` to other modules_
> _refactoring in spring_

### Walkthrough
*  Export `SpaceProxy` class from `@dxos/client` package to enable interaction with spaces from other modules ([link](https://github.com/dxos/dxos/pull/3724/files?diff=unified&w=0#diff-5fb344208cd3ad040e48fba4b60b2e982f0fd977d802dba856fcf5a45115ab36R9)).

### CODE REVIEW

**Comment by mykola-vrmchk** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3724** <a href="https://app.graphite.dev/github/pr/dxos/dxos/3724" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dxos/dxos/3724?utm_source=stack-comment).

**Comment by mykola-vrmchk** [comment]
Fix for https://github.com/dxos/dxos/issues/3713

File: packages/apps/labs-app/src/main.tsx [hunk 1]
```diff
@@ -22,10 +22,18 @@ import { ThemePlugin } from '@braneframe/plugin-theme';
 import { ThreadPlugin } from '@braneframe/plugin-thread';
 import { TreeViewPlugin } from '@braneframe/plugin-treeview';
 import { UrlSyncPlugin } from '@braneframe/plugin-url-sync';
+import { SpaceProxy } from '@dxos/client';
```
Comment (wittjosiah) [inline packages/apps/labs-app/src/main.tsx:25]:
```suggestion
import { SpaceProxy } from '@dxos/client/echo';
```

Comment (mykola-vrmchk) [reply packages/apps/labs-app/src/main.tsx]: thx

**Pull request merged**